src/System.Management.Automation/engine/parser/PSType.cs

```diff
@@ -542,7 +542,7 @@ public void DefineMembers()
 
             private void DefineProperty(PropertyMemberAst propertyMemberAst)
             {
-                if (_definedProperties.ContainsKey(propertyMemberAst.Name))
+                if (_definedProperties.ContainsKey(propertyMemberAst.Name) || _definedMethods.ContainsKey(propertyMemberAst.Name) && !_definedMethods[propertyMemberAst.Name].Any(m => m.Item1.IsConstructor))
```

Comment:
```suggestion
                if (_definedProperties.ContainsKey(propertyMemberAst.Name) || (_definedMethods.ContainsKey(propertyMemberAst.Name) && !_definedMethods[propertyMemberAst.Name].Any(m => m.Item1.IsConstructor)))
```

Reply:
Already fixed 😅

Reply:
Could we avoid LINQ?

Reply:
Actually we can avoid LINQ altogether - `IsConstructor` is name-bound, so if any `m` satisfies `IsConstructor == true`, they all will. We can simply inspect the first item in the list